src/ui/components/NodePickerContext.tsx

```diff
@@ -28,7 +28,7 @@ export type NodePickerOptions = {
 };
 
 export type NodePickerContextType = {
-  enable: (options: NodePickerOptions, initializer: () => Promise<void>) => Promise<void>;
+  enable: (options: NodePickerOptions, initializer?: () => Promise<void>) => Promise<void>;
```

Comment:
Is there any value in keeping the initializer param? The only place calling this function has been updated so as not to pass it.